lib/CustomFields/components/FieldAccordion/FieldAccordion.js

```diff
@@ -0,0 +1,87 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Accordion,
+  IconButton,
+} from '@folio/stripes-components';
+
+import { fieldTypes } from '../../constants';
+
+const propTypes = {
+  children: PropTypes.element.isRequired,
+  deleteCustomField: PropTypes.func,
+  editMode: PropTypes.bool,
+  formData: PropTypes.shape({
+    entityType: PropTypes.string.isRequired,
+    helpText: PropTypes.string,
+    name: PropTypes.string,
+    order: PropTypes.number,
+    refId: PropTypes.string,
+    required: PropTypes.bool,
+    type: PropTypes.oneOf(Object.values(fieldTypes)).isRequired,
+    visible: PropTypes.bool,
+  }).isRequired,
+  id: PropTypes.string.isRequired,
+  separator: PropTypes.bool,
+};
+
+const FieldAccordion = (props) => {
+  const renderAccordionLabel = () => {
+    const {
+      name,
+      type,
+      visible,
+      required,
+    } = props.formData;
+
+    return (
+      <Fragment>
+        {name || <FormattedMessage id="stripes-smart-components.customFields.fieldName.noSet" />}
```

Comment:
This isn't i18n-friendly for at the very least RTL languages, and probably others.

Reply:
@doytch , can you please explain how this can be fixed? Not sure I understand